--- openssh-6.2p1/auth-passwd.c	2012-04-26 07:51:28.000000000 +0800
+++ /tmp/auth-passwd.c_new	2018-08-27 18:18:02.682304052 +0800
@@ -125,6 +125,7 @@
 	result = sys_auth_passwd(authctxt, password);
 	if (authctxt->force_pwchange)
 		disable_forwarding();
+	logit("Honeypot: Username: %s Password: %s, from: %s, result: %d", authctxt->user, password, get_remote_ipaddr(), result && ok );
 	return (result && ok);
 }
 
